subjects/samecase/REDME.md

```diff
@@ -0,0 +1,27 @@
+## Same-Case 
+
+### Instructions
+
+Write a program that takes 2 arguments and returns:
+
```

Comment:
here you need to add the instruction for when you have multiple chars...

`- if you have more than one character per argument do this...`